### PR TITLE
add support for measure observation on measure group, with stubbed va…

### PR DESCRIPTION
**Pull request by nmorasb**
…lidator to be implemented later

## MADiE PR Template

Jira Ticket: [MAT-4525](https://jira.cms.gov/browse/MAT-4525)
(Optional) Related Tickets:

### Summary
Initial PR to add measure observation
Validators and test for validators will be added in a subsequent PR

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e. Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by adongare — DISMISSED** [review]

File: src/main/java/gov/cms/madie/models/measure/MeasureObservation.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      groups = {Measure.ValidationOrder5.class}
+  )
+  private String definition;
+  private String criteriaReference;
+  @NotNull(
+      message = "Aggregate Method is required.",
+      groups = {Measure.ValidationOrder5.class})
+  @EnumValidator(
+      enumClass = AggregateMethodType.class,
+      message = "Aggregate Method must be a valid MADiE Aggregate Method type",
```
Comment (adongare) [inline src/main/java/gov/cms/madie/models/measure/MeasureObservation.java:29]: We may not call it MADiE aggregate method type.

File: src/main/java/gov/cms/madie/models/validators/MeasureObservationValidator.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@Slf4j
+public class MeasureObservationValidator implements ConstraintValidator<ValidMeasureObservation, Group> {
+  @Override
+  public boolean isValid(Group value, ConstraintValidatorContext context) {
+    // TODO - fill this in based on the following logic
+    // If an observation exists, it must only be for CV or Ratio scoring types.
+    // CV can have only one measure observation
+    // Ratio can have only two, and must reference the NUMER or DENOM populations (cannot have two referencing same population)
+    return true;
+  }
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/models/validators/MeasureObservationValidator.java:18]: Do we have a future story to implement this?

**Review by RohitKandimalla — DISMISSED** [review]

**Review by adongare — APPROVED** [review]
